bigquery/tests/unit/test_external_config.py

```diff
@@ -56,15 +55,64 @@ def test_api_repr_base(self):
                 ],
             },
         })
-        ec = ExternalConfig.from_api_repr(resource)
+        ec = external_config.ExternalConfig.from_api_repr(resource)
         self._verify_base(ec)
-        self.assertEqual(ec.schema,
-                         [SchemaField('full_name', 'STRING', mode='REQUIRED')])
+        exp_schema = [
+            schema.SchemaField('full_name', 'STRING', mode='REQUIRED')
+        ]
+        self.assertEqual(ec.schema, exp_schema)
         self.assertIsNone(ec.options)
 
         got_resource = ec.to_api_repr()
         self.assertEqual(got_resource, resource)
 
+    def test_to_api_repr_base(self):
+        ec = external_config.ExternalConfig('')
+        ec.source_uris = self.SOURCE_URIS
+        ec.max_bad_records = 17
+        ec.autodetect = True
+        ec.ignore_unknown_values = False
+        ec.compression = 'compression'
+        ec.schema = [
+            schema.SchemaField('full_name', 'STRING', mode='REQUIRED')
+        ]
+
+        exp_schema = {
+            'fields': [
+                {
+                    'name': 'full_name',
+                    'type': 'STRING',
+                    'mode': 'REQUIRED',
+                    'description': None,
+                },
+            ]
+        }
+        got_resource = ec.to_api_repr()
+        exp_resource = {
+            'sourceFormat': '',
+            'sourceUris': self.SOURCE_URIS,
+            'maxBadRecords': 17,
+            'autodetect': True,
+            'ignoreUnknownValues': False,
+            'compression': 'compression',
+            'schema': exp_schema
+        }
+        self.assertEqual(got_resource, exp_resource)
+
+    def test_schema_None(self):
```

Comment:
I guess this passes flake8?

Reply:
I'm not sure what you mean by that. Is it a bad name? Yes everything is passing.

Reply:
Yeah, it's an odd name, since it wouldn't be valid in a regular function to use a capital letter. But I think I've seen tests use `..._None` before, so I'm fine so long as `flake8` is also fine with it.